api/parts/data/fetch.js

```diff
@@ -1827,4 +1828,89 @@ function union(x, y) {
     return res;
 }
 
+/**
+* Get data for jobs listing for jobs api
+* @param {string} metric - name of the collection where to get data from
+* @param {params} params - params object with app_id and date
+*/
+fetch.fetchJobs = async function(metric, params) {
+    if (params.qstring.name) {
+        fetch.jobDetails(metric, params);
+    }
+    else {
+        fetch.alljobs(metric, params);
+    }
+};
+
+/**
+* Get all jobs grouped by job name for jobs api
+* @param {string} metric - name of the collection where to get data from
+* @param {params} params - params object with app_id and date
+*/
+fetch.alljobs = async function(metric, params) {
+    const columns = ["name", "schedule", "next", "finished", "status", "total"];
+    let sort = {};
+    let total = await countlyDb._native.collection('jobs').aggregate([
+        {
+            $group: { _id: "$name" }
+        },
+        {
+            $count: 'total'
+        }
+    ]).toArray();
+    total = total.length > 0 ? total[0].total : 0;
+    const cursor = countlyDb._native.collection('jobs').aggregate([
```

Comment:
Codacy found an issue: ['countlyDb' is not defined.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5834029)